compiler/src/iree/compiler/Codegen/LLVMGPU/LLVMGPUVectorUnrolling.cpp

```diff
@@ -0,0 +1,94 @@
+// Copyright 2024 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
```

Comment:
Did you mean to include this file, seems to be duplication from LLVMGPUVectorLowering.cpp